services/document-manager/backend/app/__init__.py

```diff
@@ -24,12 +16,40 @@
 
 from celery import Celery
 
+from opentelemetry import trace
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.instrumentation.flask import FlaskInstrumentor
+from logging.config import dictConfig
+
 
-def create_app(config_object=None):
+def create_app(test_config=None):
     """Create and configure an instance of the Flask application."""
+    if test_config is None:
+        dictConfig(Config.LOGGING_DICT_CONFIG)
     app = Flask(__name__)
 
-    config = config_object if config_object else Config
+    trace.set_tracer_provider(TracerProvider())
+
+    FlaskInstrumentor().instrument_app(app)
+
+    @app.after_request
+    def log_response_info(response):
+        from datetime import datetime
+        def get_formatted_current_time():
+            return datetime.now().strftime("%d/%b/%Y %H:%M:%S")
```

Comment:
I think that the datetime helper function imported from core-api should probably also go into a helper/utils file, would also prefer to avoid imports within functions.